### ISSUE TITLE
Villagers still way too happy

### ISSUE THREAD

**Issue by Magleose** [TangoTek/TekTopia-Community#578]
**Describe the bug**
I take night time in game to check on individual villagers and see what is going on with their inventory and stats. I was working on doing this when I opened one of the houses there were three hearts on the ground. I checked all the villagers at that point and most of them save for 3-4 were all 100% happy. I have in the neighborhood of 30ish villagers. This is in the most recent jar, 1.0.49

**To Reproduce**
1. have mature village with all facilities running
2. wait for night time
3. check villager happiness.

**Expected behavior**
I am not expecting a  downtrodden workforce or anything but I expect around 80-85% happiness, with the last 15-20% hard to attain. Like villagers should have just the best day ever to get to 100% happiness.  It feels like a weird thing to make a bug report over but it does not feel balanced as is.

**Screenshots**
![2019-06-29_15 20 55](https://user-images.githubusercontent.com/37353615/60388651-b97e5100-9a82-11e9-80eb-bb3dca0e3abd.png)

** Server/Location **
Server: Main(Survival) 2
Town Hall Marker: 1357 / 70 / 5261
City Name: Bonitos Pantelones

**Comment by Sobertron** [comment]
So i am having the opposite problem. Before .49 none of my villagers (over 200) were ever unhappy. After .49 I am noticing them, mostly miners. It also does not help that our bards do not appear to be playing in the tavern at night. When all the other villagers go to the Tavern, the bards seem to lose their AI and will stop here they are. They only move again when it is time to go to bed.

**Comment by Magleose** [comment]
yeah, now that you mention it, the ones that were not 100% happy were miners, and my bard has not been in the tavern for the most part

**Comment by paspinall42** [comment]
and yet I am getting maybe 1 heart every 3 nights
